frontend/src/metabase/admin/databases/database.js

```diff
@@ -283,6 +286,20 @@ export const discardSavedFieldValues = createThunkAction(
   },
 );
 
+export const persistDatabase = createThunkAction(
+  PERSIST_DATABASE,
+  databaseId => async () => {
+    await MetabaseApi.db_persist({ dbId: databaseId });
+  },
+);
+
+export const unpersistDatabase = createThunkAction(
+  UNPERSIST_DATABASE,
+  databaseId => async () => {
+    await MetabaseApi.db_persist({ dbId: databaseId });
```

Comment:
I'm surprised to see the persist and unpersist both call `MetabaseApi.db_persist({ dbId: databaseId })` i would have expected them to be a little different. Am i missing something? I see the first arg to create ThunkAction is a little different.

Reply:
Thanks, fixed!